collection-mocks-tests.js

```diff
@@ -1,39 +1,39 @@
 // Tests if mock object property gets set when using the insert argument
 Tinytest.add('mock method - insert argument - property gets set correctly', function (test) {
-  var TestCollection = new Mongo.Collection('test1');
+  var TestCollection = new Mongo.Collection(null);
```

Comment:
Using `null` as the first argument makes the collection virtual (memory only) which makes this collection more like a variable, we can use it and toss it, also we don't need a unique name. (It's better for performance as well, since we don't setup any auto-publish publications or methods for client insertion, etc.)